docs_src/tutorial/relationship_attributes/back_populates/tutorial003.py

```diff
@@ -3,6 +3,21 @@
 from sqlmodel import Field, Relationship, SQLModel, create_engine
 
 
+class Hero(SQLModel, table=True):
+    id: Optional[int] = Field(default=None, primary_key=True)
+    name: str = Field(index=True)
+    secret_name: str
+    age: Optional[int] = Field(default=None, index=True)
+
+    team_id: Optional[int] = Field(default=None, foreign_key="team.id")
+    team: Optional["Team"] = Relationship(back_populates="heroes")
+
+    weapon_id: Optional[int] = Field(default=None, foreign_key="weapon.id")
+    weapon: Optional["Weapon"] = Relationship(back_populates="hero")
+
+    powers: List["Power"] = Relationship(back_populates="hero")
```

Comment:
Why putting `Hero` before `Weapon`? Is there any functional reasoning behind? Stringed annotations feel more noisy/dirty, IMO.

Reply:
I think this was due to circular import error when testing. Stringed annotations are a must if you intend to use sqlmodel or it will fail in complex structure models from experience.

Reply:
Somewhat valid, yeah. I, too, experienced circular dependencies. Although, I had some workarounds (local imports mostly).